docs/CNVs/CNV-methods.tex

```diff
@@ -670,125 +712,170 @@ \subsection{Generative Model for Read Counts} \label{coverage-model}
 %
 \begin{align}
-\vz \sim& {\rm N}(\vzero, \vI) \\
-\vb \sim& {\rm N}(\vW \vz + \vm, \vPsi),
+\vz &\sim \norm(\vzero, \vI), \nonumber\\
+\vb &\sim \norm(\vW \vz + \vm, \vPsi),
 \end{align}
 %
-where $\vz \in \mathbb{R}^D$ is a low-dimensional latent vector of laboratory conditions, $\vW \in \mathbb{R}^{T \times D}$ is a linear map from latent space to target space, $\vm \in \mathbb{R}^T$ is the vector of mean biases, and $\vPsi \in \mathbb{R}^{T \times T}$ is a diagonal matrix of residual variance not explained by the latent features.  We approximate the Poisson as a Gaussian centered at its mode (as a function of $b_{st}$):
+where $\vz \in \mathbb{R}^D$ is a low-dimensional latent vector of laboratory conditions, $\vW \in \mathbb{R}^{T \times D}$ is a linear map from latent space to target space, $\vm \in \mathbb{R}^T$ is the vector of mean biases, and $\vPsi \in \mathbb{R}^{T \times T}$ is a diagonal matrix of residual variance not explained by the latent features. We approximate the Poisson as a Gaussian (an excellent approximation for $n_{st} \gtrsim 1000$) and expand the argument of the Gaussian exponential about the mode of $b_{st}$ to quadratic order to obtain:
 %
+\begin{equation}\label{eq:gaussian_approx_coverage}
+{\rm Poisson}(n_{st} | d_s c_{st} e^{b_{st}}) \simeq \Sigma_{st} \, \norm(b_{st} | m_{st}, \Sigma_{st}),
+\end{equation}
+where:
 \begin{align}
-\prod_t {\rm Poisson}(n_{st} | d_s c_{st} e^{b_{st}}) \approx {\rm N}(\vb_s | \vm_s, \vSigma_s), \, m_{st} = \ln(n_{st}/(c_{st} d_s)), \, \Sigma_{st} = 1/n_{st}
+m_{st} &= \ln(n_{st}/(\PP_{st} c_{st} d_s)), \nonumber\\
+\Sigma_{st} &= 1/n_{st}.
 \end{align}
-%
-to obtain the joint likelihood
-%
+
+\begin{figure}
+\center
+\includegraphics[scale=0.7]{figs/{gauss_poisson_0.1}.pdf}
+\includegraphics[scale=0.7]{figs/{gauss_poisson_10}.pdf}
+\caption{Gaussian approximation to the Poisson likelihood (see Eq.~\ref{eq:gaussian_approx_coverage}). The left and right panels show $\mathrm{Poisson}(n|\alpha\,e^b)$ and $n^{-1}\mathcal{N}(b|\ln(n/\alpha), n^{-1})$, respectively for $\alpha=0.1$ (top) and $\alpha=10.0$ (bottom). The black lines show $b = \ln(n/\alpha)$ the maximum likelihood bias estimate. The Gaussian approximation breaks down at $n=0$ (no coverage). It also slightly overestimates the variance at small $n$. Otherwise, it is an excellent approximation.}
```

Comment:
Might also be nice to plot the ratio or the relative error between the two quantities.